modules/virt-creating-template-wizard-web.adoc

```diff
@@ -10,27 +10,44 @@ The web console features an interactive wizard that guides you through the *Gene
 All required fields are marked with a `*`. The wizard prevents you from moving to the next step
 until you provide values in the required fields.
 
+[NOTE]
+====
+The interactive wizard guides you to create a custom virtual machine template where you specify the operating system, boot source, flavor, and other settings. You cannot select a preconfigured Red Hat template when you use the interactive wizard.
```

Comment:
```suggestion
The interactive wizard guides you to create a custom virtual machine template where you specify the operating system, boot source, flavor, and other settings.
```
I was able to use preconfigured Red Hat template, what am I missing ?

Reply:
Removed the sentence.